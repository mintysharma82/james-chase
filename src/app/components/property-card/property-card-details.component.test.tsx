import { render } from "@testing-library/react";
import SearchCardDetails from "./property-card-details.component";
import React from "react";

describe("Component > Property Card > Details", () => {
  it("renders with rating as expected", () => {
    const mockHolidayData = {
      content: {
        hotelFacilities: ["Restaurant", "Swimming Pool"],
        starRating: "5",
        atAGlance: [],
      },
    };
    const { container } = render(
      <SearchCardDetails
        boardBasis={"Room"}
        atAGlance={["My Feature"]}
        hotelFacilities={["Restaurant", "Swimming Pool"]}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
