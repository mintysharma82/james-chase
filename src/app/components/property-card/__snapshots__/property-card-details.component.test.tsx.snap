// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component > Search Result > Search Card Details renders with rating as expected 1`] = `
<div>
  <div>
    <p>
      Room
    </p>
    <ul
      class="sc-11f3f108-4 eBwZep"
    >
      <li>
        My Feature
      </li>
    </ul>
    <h3>
      Amenities
    </h3>
    <ul
      class="sc-11f3f108-4 eBwZep"
    >
      <li>
        Restaurant
      </li>
      <li>
        Swimming Pool
      </li>
    </ul>
  </div>
</div>
`;
